queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts > 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
```

Comment:
- [ ] Ouch! the animal should have less than 3 escape attempts but you seem to have reversed in the query at line `5`.

Reply:
Okay, thank you